docs/source/configuration.rst

```diff
@@ -650,6 +650,48 @@ Developer configuration options
     Time in seconds to wait after persistent volume gets initialized to prevent
     mount errors on BLAST kubernetes jobs.
 
+.. _elb_disable_janitor:
+
+``ELB_DISABLE_AUTO_SHUTDOWN``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    **Set via an environment variable**.
+
+    * Default: Disabled
+    * Values: Any string. Set to any value to enable.
+
+    Disables the auto-shutdown feature.
+
+.. _elb_janitor_schedule:
+
+``ELB_JANITOR_SCHEDULE``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    **Set via an environment variable**.
+
+    * Default: run every 5 minutes.
+
+      * For GCP: ``*/5 * * * *``
+      * For AWS: ``cron(*/5 * * * ? *)``
```

Comment:
Would not it be better to unify this? Make this parameter a source for cron(x) for AWS.

Reply:
@victzh  - Can you please elaborate on to unify this?

Reply:
To have */x * * * * for both GCP and AWS, but in AWS wrap user set value in cron(x). Do you see any usefulness in setting AWS parameter to anything else than cron?

Reply:
Thanks for the clarification, yes, I agree that that would be useful and potentially safer. Leaving it as is leave us more flexibility for different scheduling configurations (e.g.: one off, daily, etc).